Directory.Build.props

```diff
@@ -125,6 +125,11 @@
     <SymbolsOutputPath>$(ArtifactsDir)symbols\$(Configuration)\</SymbolsOutputPath>
   </PropertyGroup>
 
+  <!-- Include our "loose" PDBs when publishing. -->
+  <ItemGroup>
+    <FilesToPublishToSymbolServer Include="$(SymbolsOutputPath)**\*.pdb" />
+  </ItemGroup>
```

Comment:
This could also be placed in Publishing.props now. Let's do that if we need another iteration.